packages/services/pkg/mode/storecore/encoding.go

```diff
@@ -743,52 +748,60 @@ func (schemaType SchemaType) ToSolidityType() string {
 	if strings.Contains(schemaType.String(), "ARRAY") {
 		_type := strings.Split(schemaType.String(), "_")[0]
 		return fmt.Sprintf("%s[]", strings.ToLower(_type))
-	} else {
-		return strings.ToLower(schemaType.String())
 	}
+	return strings.ToLower(schemaType.String())
 }
 
 // SchemaTypeToPostgresType converts the specified SchemaType instance to the corresponding PostgreSQL type.
 // The function returns a string representing the PostgreSQL type for the specified SchemaType instance.
 //
 // Returns:
 // (string) - A string representing the PostgreSQL type for the specified SchemaType instance.
+//
+//nolint:gocognit // auto-gen + readablity
 func (schemaType SchemaType) ToPostgresType() string {
+	const text = "text"
+	const jsonb = "jsonb"
+	const boolean = "boolean"
+
+	//nolint:gocritic,nestif // no switch, simplify
 	if (schemaType >= UINT8 && schemaType <= UINT32) || (schemaType >= INT8 && schemaType <= INT32) {
 		// Integer.
-		return "integer"
+		return text
```

Comment:
does this mean we can't query with numbers anymore?
If I want to query for an "area" (ie: X >= X_START & X < X_END & Y >= Y_START & Y < Y_END), does this mean this change would make this query impossible (or impossibly slow)?